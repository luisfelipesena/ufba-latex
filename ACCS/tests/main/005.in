2977 32452831 58337876
