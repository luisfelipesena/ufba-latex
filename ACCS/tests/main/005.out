1893229230729933
